cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1030,5 +1030,24 @@ function Verify-WorkerServices {
   Log_Todo "run more verification commands."
 }
 
+# Add a registry key for docker in EventLog so that log messages are mapped
+# correctly.
```

Comment:
Add a link to where you found this solution?

Reply:
Done